packages/neos-ui-redux-store/src/UI/LeftSideBar/index.ts

```diff
@@ -8,13 +8,19 @@ export interface State extends Readonly<{
     isHidden: boolean;
     contentTree: {
         isHidden: boolean;
+    },
+    searchBar:  {
+        searchToggled: boolean;
     };
 }> {}
 
 export const defaultState: State = {
     isHidden: false,
     contentTree: {
         isHidden: false
+    },
+    searchBar: {
+        searchToggled: false
```

Comment:
Wouldn't `isHidden` fit better with the other state variables and be more explicit.
From `searchToggled` I wouldn't easily know which state means what.

Reply:
Had this before ;)

Reply:
The thing is that it's visible by default. Maybe 'searchShown' or 'searchVisible'?

Reply:
I dunno, maybe 'isHidden' would also work

Reply:
`isVisible` is also fine.